src/implementations/config/create.ts

```diff
@@ -0,0 +1,211 @@
+import {
+  Issuer,
+  RevocationType,
+  IdentityProofType,
+  OpenAttestationDocument,
+} from "@govtechsg/open-attestation/dist/types/__generated__/schema.2.0";
+import fs from "fs";
+import path from "path";
+import { info, success } from "signale";
+import { deployDocumentStore } from "../../implementations/deploy/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as createWallet } from "../../implementations/wallet/create";
+import tradetrustConfig from "./templates/tradetrust.json";
+import { highlight } from "../../utils";
+import { handler as createTemporaryDns } from "../../commands/dns/txt-record/create";
+import { CreateConfigCommand } from "../../commands/config/config.type";
+
+interface ConfigFile {
+  wallet: EncryptedWallet | AwsKmsWallet;
+  forms: Form[];
+}
+
+interface EncryptedWallet {
+  type: "ENCRYPTED_JSON";
+  encryptedJson: string;
+}
+
+interface AwsKmsWallet {
+  type: "AWS_KMS";
+  accessKeyId: string;
+  region: string;
+  kmsKeyId: string;
+}
+
+interface Form {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  defaults: OpenAttestationDocument;
+}
+
+interface TypesOfForms {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  identityProofTypes: (IdentityProofType | undefined)[];
+}
+
+const sandboxEndpointUrl = "https://sandbox.fyntech.io";
+
+export const create = async ({
+  encryptedWalletPath,
+  outputDir,
+  configType,
+  configTemplatePath,
+}: CreateConfigCommand): Promise<string> => {
+  let walletPath = "";
+  if (!encryptedWalletPath) {
+    info(`Wallet file not provided, please enter password to create a new wallet`);
+    const createWalletParams = {
+      outputFile: path.join(outputDir, "wallet.json"),
+      fund: "ropsten",
+    };
+    walletPath = await createWallet(createWalletParams);
+    info(`Wallet created at ${walletPath}`);
+  }
+  const walletFilePath = walletPath || encryptedWalletPath;
```

Comment:
able to move this out as a separate function `getWalletPath` maybe?

Reply:
ok